package.json

```diff
@@ -25,11 +25,12 @@
   },
   "homepage": "https://github.com/cronofy/cronofy-node#readme",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "rest": "^1.3.1"
+    "lodash": "^4.17.4",
```

Comment:
Say I've got some other dependency that `"lodash": "^3.0.0"`, if we only say `^4.0.0` then they cannot get the latest improvements to this library as the dependency restrictions are incompatible.

If we loosen our restrictions to all the "known good" versions then we'll play nicely with a larger combination of dependencies.